src/js/components/user/githubUser.tsx

```diff
@@ -162,24 +168,68 @@ export const AssignUsersModal = ({
           onClick={handleSubmit}
         />,
       ]}
+      size="small"
       onRequestClose={handleClose}
     >
-      <DataTable
-        className="align-checkboxes table-row-targets"
-        items={allUsers}
-        selectRows="checkbox"
-        selection={selection}
-        onRowChange={updateSelection}
+      <div
+        className="slds-grid
+          slds-grid_vertical-align-start
+          slds-p-around_medium"
       >
-        <DataTableColumn
-          label={i18n.t('GitHub Users')}
-          property="login"
-          primaryColumn
-          truncate
+        <div className="slds-grid slds-wrap slds-shrink slds-p-right_medium">
+          <p>
+            <Trans i18nKey="projectCollaborators">
+              Only users who have access to the GitHub repository for this
+              project will appear in the list below. Visit GitHub to invite
+              additional collaborators to this repository.
+            </Trans>
+          </p>
+        </div>
+        <div
+          className="slds-grid
+            slds-grow
+            slds-shrink-none
+            slds-grid_align-end"
+        >
+          {isRefreshing ? (
+            <Button
+              label={
+                <LabelWithSpinner label={i18n.t('Syncing Collaborators…')} />
+              }
+              variant="outline-brand"
+              disabled
+            />
+          ) : (
+            <Button
+              label={i18n.t('Re-Sync Collaborators')}
+              variant="outline-brand"
+              iconCategory="utility"
+              iconName="refresh"
+              iconPosition="left"
+              onClick={refreshUsers}
+            />
+          )}
+        </div>
+      </div>
+      <div className="slds-is-relative">
+        <DataTable
```

Comment:
I think we need an empty state for this list?

![empty list looks broken](https://dl.dropbox.com/s/27xwrgum9pt3h5e/Screenshot%202020-02-04%2015.40.40.jpg)

Reply:
Yeah... That _should_ never be possible, since if you yourself are not in that list then you should've even be able to use MetaShare at all... but not a bad idea to add a dummy fallback.

Reply:
@mirisuzanne Added empty state:

<img width="690" alt="Screenshot 2020-02-05 08 53 34" src="https://user-images.githubusercontent.com/552316/73847822-04203280-47f5-11ea-9b83-fd2db29a1bb1.png">